src/devices/Board/PwmChannelManager.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.Pwm;
+
+namespace Iot.Device.Board
+{
+    internal class PwmChannelManager : PwmChannel
```

Comment:
I'm confused about this class. The name says manager but yet it implements PwmChannel